### PR TITLE
Ticket 94: Reason about fetching of inclusion proofs.

### PR DESCRIPTION
**Pull request by eranmes**
This change moves the two paragraphs about inclusion proof fetching and
validation together.
It also provides motivation for fetching inclusion proofs and notes that
direct fetching of inclusion proofs discloses private information to the
log.

### CODE REVIEW

**Comment by robstradling** [comment]
Other than that, LGTM.

File: rfc6962-bis.xml [hunk 1]
```diff
@@ -1293,10 +1293,10 @@ but it is expected there will be a variety.
...
         </t>
         <t>
-          To be considered compliant, a certificate MUST be accompanied by at least one valid SCT or at least one valid inclusion proof. A certificate not accompanied by any valid SCTs or any valid inclusion proofs MUST NOT be considered compliant by TLS clients. However, specifying the TLS clients' behavior once compliance or non-compliance has been determined (for example, whether a certificate should be rejected due to non-compliance) is outside the scope of this document.
+          TLS clients SHOULD also validate each supplied inclusion proof (see <xref target="verify_inclusion"/>), to audit the log and gain confidence that the certificate is logged. If no inclusion proof was supplied, the TLS client MAY request one directly from the corresponding log using <spanx style="verb">get-proof-by-hash</spanx> (<xref target="get-proof-by-hash"/>) or <spanx style="verb">get-all-by-hash</spanx> (<xref target="get-all-by-hash"/>), and then validate it. Note that a direct request from the log discloses to the log which TLS server the client has been communicating with.
```
Comment (benlaurie) [inline rfc6962-bis.xml]: s/validate/verify/.
s/direct request from/direct request to/

**Comment by benlaurie** [comment]
LGTM other than comment (Rob's proposed alternative language works for me, too).

File: rfc6962-bis.xml (hunk 1 above)
Comment (eranmes) [inline rfc6962-bis.xml]: @benlaurie I assume you meant s/validate/verify/g as there were two instances of 'validate' in this sentence.

**Comment by eranmes** [comment]
Used Rob's alternative language as it's more terse.
PTAL.

**Pull request merged**